public/app/features/library-panels/components/OpenLibraryPanelModal/OpenLibraryPanelModal.tsx

```diff
@@ -38,38 +39,46 @@ export function OpenLibraryPanelModal({ libraryPanel, onDismiss }: OpenLibraryPa
   };
 
   return (
-    <Modal title="View panel in dashboard" onDismiss={onDismiss} onClickBackdrop={onDismiss} isOpen>
+    <Modal
+      title={t('library-panels.modal.title', 'View panel in dashboard')}
+      onDismiss={onDismiss}
+      onClickBackdrop={onDismiss}
+      isOpen
+    >
       <div className={styles.container}>
         {connected === 0 ? (
-          <span>Panel is not linked to a dashboard. Add the panel to a dashboard and retry.</span>
+          <span>
+            <Trans i18nKey={'library-panels.modal.panel-not-linked'}>Add the panel to a dashboard and retry.</Trans>
```

Comment:
this `<Trans>` tag is missing the first part of the translation string (`Panel is not linked to a dashboard.`)

Reply:
Can't believe I missed that.
Just updated the PR @ashharrison90